pkg/policies/dataplane/actuators/rate/rate-limiter.go

```diff
@@ -355,3 +355,9 @@ func (rateLimiter *rateLimiter) GetLimiterID() iface.LimiterID {
 		PolicyHash:     rateLimiter.GetPolicyHash(),
 	}
 }
+
+// GetHistogram is there to satisfy Limiter interface.
+func (rateLimiter *rateLimiter) GetHistogram(map[string]string) prometheus.Observer {
```

Comment:
FYI, would need metrics for RateLimiter too: https://github.com/fluxninja/aperture/issues/459

At least a count metric will be required. Can do a latency summary as well

Reply:
Ok, I'll take this task from Hasit